backend/data_validation/data_val_2022.py

```diff
@@ -126,3 +127,85 @@ def check_for_missing_shooting_zones(
                 f"In {match_key}, {team_number} MISSING TELEOP SHOOTING ZONES",
                 error_type=ErrorType.MISSING_DATA,
             )
+
+    def check_for_statistical_outliers(
+        self,
+        auto_check: bool = True,
+        teleop_check: bool = True,
+        endgame_check: bool = True,
+    ) -> None:
+        """
+        Checking for statistical outliers in the 2022 Rapid React game.
+
+        :param auto_check: Runs outlier checks specifically and strictly against the autonomous portion of the game.
+        :param teleop_check: Runs outlier checks specifically and strictly against the teloperated portion of the game.
+        :param endgame_check: Runs outlier checks specifically and strictly against the endgame portion of the game.
+
+        :return:
+        """
+        self.check_for_auto_outliers()
+
+    def check_for_auto_outliers(self) -> None:
+        """
+        Check and mark any statistical outliers across all teams' auto data.
+
+        :return:
+        """
+        for team in self.teams:
+            team_data_entries = self.df.loc[self.df["team_number"] == team]
+
+            auto_cargo_match_team_data = list(
+                zip(
+                    team_data_entries["match_key"],
+                    team_data_entries["auto_lower_hub"],
+                    team_data_entries["auto_upper_hub"],
+                    team_data_entries["taxied"],
+                )
+            )
+
+            auto_points_match_team_data = list(
+                map(
+                    lambda team_datum: (
+                        team_datum[0],
+                        team_datum[1] * Constants.RapidReact.AUTO_LOWER_HUB_POINT_VALUE
+                        + team_datum[2]
+                        * Constants.RapidReact.AUTO_UPPER_HUB_POINT_VALUE
+                        + team_datum[3] * Constants.RapidReact.AUTO_TAXI_POINT_VALUE,
```

Comment:
The formatting of this is a little off -- we have Line 170 on one line, but multiplying team_datum[2] and the AUTO_UPPER_HUB_POINT_VALUE is on two different lines, we should probably keep them on both one line.